ogre2/src/media/materials/programs/heat_signature_fs.glsl

```diff
@@ -35,12 +35,16 @@ out vec4 fragColor;
 uniform float minTemp = 0.0;
 uniform float maxTemp = 100.0;
 
+uniform int bitDepth;
+uniform float resolution;
+
 // map a temperature from the [0, 655.35] range to the [minTemp, maxTemp]
-// range (655.35 is the largest Kelvin value allowed)
+// range (655.35 is the largest Kelvin value allowed for 16 bit with
+// 10mK resolution)
```

Comment:
Similar to my other comment about possible confusion when referring to 10mK resolution and 16 bit format, perhaps we should change this comment to say something like `map a temperature from the camera's [min,max] range to the user-defined [minTemp,maxTemp] range`

Reply:
43c80a9